packages/jest-cli/src/test_scheduler.js

```diff
@@ -96,6 +97,24 @@ export default class TestScheduler {
         });
         return Promise.resolve();
       }
+
+      // **EXPERIMENTAL!!** Throws when the context is leaked after executing
+      // a test.
+      if (testResult.leaks) {
+        const message = `Your test suite is leaking memory! Please ensure all references are cleaned.
```

Comment:
Wait, the idea was to include the experimental text in the error message, not in a comment above the error message. The user should see this is experimental right now.

Reply:
😂 OK, my bad; I thought you wanted me to add it as a comment so that when people looks why it broke, they will realize it's experimental. I will add it to the message.

Reply:
Yeah, rm the comment :D